x-pack/plugins/observability/public/components/app/cases/translations.ts

```diff
@@ -92,10 +92,6 @@ export const OPTIONAL = i18n.translate('xpack.observability.cases.caseView.optio
   defaultMessage: 'Optional',
 });
 
-export const PAGE_TITLE = i18n.translate('xpack.observability.cases.pageTitle', {
-  defaultMessage: 'Cases',
-});
-
```

Comment:
Please remove this string from the translation files `x-pack/plugins/translations/translations/zh-CN.js` and `x-pack/plugins/translations/translations/ja-JP.js`, which are causing a [failure in CI checks](https://buildkite.com/elastic/kibana-pull-request/builds/5869#b521aa78-9b51-485b-bd4d-684006541165)

Reply:
When you remove a string from the translation files you also need to run `node scripts/i18n_check.js --fix`

Reply:
Here's how I handle translation changes https://github.com/elastic/kibana/pull/116446#discussion_r738268227

Reply:
Thanks!

Reply:
Thanks. I ran `node scripts/i18n_check.js --fix`